CODEOWNERS

```diff
@@ -89,3 +89,15 @@
 /doc/ref_impl/cntt-ri2/chapters/chapter07.md        @rabi-abdel @tomkivlin
 /doc/ref_impl/cntt-ri2/chapters/chapter08.md        @rabi-abdel @tomkivlin
 /doc/ref_impl/cntt-ri2/chapters/chapter09.md        @rabi-abdel @tomkivlin
+
+# Reference Conformance 2
+/doc/ref_cert/RC2/chapters/chapter01.md        @xmulligan @tomkivlin
```

Comment:
@xmulligan current process requires both WSL and TSL to approve a PR for merging. for consistency, would you mind having you and I (rabi-abdel) in the CODEOWNERS file.

Reply:
done